deploy/kubernetes/dolphinscheduler/templates/secret-external-etcd-ssl.yaml

```diff
@@ -0,0 +1,30 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+{{- if .Values.etcd.enabled }}
+apiVersion: v1
+kind: Secret
```

Comment:
etcd being enabled doesn't indicate ssl is enabled too, so the condition `{{- if .Values.etcd.enabled }}` to create this `Secret` doesn't seem to make sense

Reply:
Ok, I will add a new var named ssl.enable